<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GNUClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_req">
      <property name="text">
       <string>Запросы ввода</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Префикс id</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ответы сервера</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="13">
     <widget class="QGroupBox" name="Commands">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Команды</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_26">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>№26</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Выдать управляющее воздействие</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>C - ЦВМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T - ТВМ, ТВУ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P - КЦП</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>V - ЦВМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X - ТВМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y - ТВМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Z - ТВМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D - УС17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>E - УС17</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="comboBox_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="plainText_req">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Классификатор id</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="5">
     <widget class="QPushButton" name="btn_connecting">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="8">
     <widget class="QPlainTextEdit" name="plainTextEdit_inputData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="pushButton_Enter">
      <property name="text">
       <string>Ввод</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>S - СМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N - МИМ1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>H - МИМ2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M - МЛМ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>R - НЭМ</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_enter">
      <property name="text">
       <string>Поле ввода</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="1" column="5" rowspan="8">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="8">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLineEdit" name="plainText_enter"/>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLineEdit" name="plainTextEdit_class_id">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
